meteor/client/ui/Settings/components/ConfigManifestEntryComponent.tsx

```diff
@@ -61,6 +62,14 @@ export const renderEditAttribute = (
 				}
 			/>
 		)
+	// } else if (configField.type === ConfigManifestEntryType.TABLE) {
```

Comment:
I've commented out these changes, as I wasn't sure on the best ways to address these